### PR TITLE
Move away from using quay.io for default images

### PR DESCRIPTION
**Pull request by rnorth**

### CODE REVIEW

File: core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
 import lombok.extern.slf4j.Slf4j;
 import org.testcontainers.UnstableAPI;
 
-import java.io.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
```
Comment (rnorth) [inline core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java:19]: 🤔 
Will revert these imports.

File: core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java [hunk 2]
```diff
@@ -72,11 +72,11 @@ public boolean isRyukPrivileged() {
     }
 
     public String getRyukImage() {
-        return (String) properties.getOrDefault("ryuk.container.image", "quay.io/testcontainers/ryuk:0.2.3");
+        return (String) properties.getOrDefault("ryuk.container.image", "testcontainersofficial/ryuk:latest");
```
Comment (bsideup) [inline core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java:75]:
```suggestion
        return (String) properties.getOrDefault("ryuk.container.image", "testcontainersofficial/ryuk:0.3.0");
```

File: docs/features/configuration.md [hunk 3]
```diff
@@ -49,13 +49,11 @@ Another possibility is to set up a registry mirror in your environment so that a
...
 !!!tip
-    Registry mirror currently only works for Docker images with image name that has no registry specified (for example, for Docker image `mariadb:10.3.6`, it works, for Docker image `quay.io/testcontainers/ryuk:0.2.3`, not).
-    Workaround is to to configure the affected Docker image in `.testcontainers.properties`.
-    For example: `ryuk.container.image = testcontainers/ryuk:0.2.3` or `ryuk.container.image = <your.docker.registry>/testcontainers/ryuk:0.2.3`
+    Registry mirror currently only works for Docker images with image name that has no registry specified (for example, for Docker image `mariadb:10.3.6`, it works, for Docker image `quay.io/something/else`, not).
 
 ## Customizing Ryuk resource reaper
 
-> **ryuk.container.image = quay.io/testcontainers/ryuk:0.2.3**
+> **ryuk.container.image = testcontainersoffical/ryuk:0.2.3**
```
Comment (bsideup) [inline docs/features/configuration.md:56]:
```suggestion
> **ryuk.container.image = testcontainersoffical/ryuk:0.3.0**
```

File: docs/features/configuration.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ Some companies disallow the usage of Docker Hub, but you can override `*.image`
 > **tinyimage.container.image = alpine:3.5**  
 > Used by Testcontainers' core
 
-> **vncrecorder.container.image = quay.io/testcontainers/vnc-recorder:1.1.0**  
+> **vncrecorder.container.image = testcontainersoffical/vnc-recorder:1.1.0**
```
Comment (slovdahl) [inline docs/features/configuration.md:35]: Should it be `testcontainersofficial`?

File: docs/features/configuration.md (hunk 3 above)
Comment (slovdahl) [reply docs/features/configuration.md:56]: Same question as above.

File: docs/features/configuration.md (hunk 4 above)
Comment (rnorth) [reply docs/features/configuration.md:35]: Whoops, quite right.

File: core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java (hunk 2 above)
Comment (rnorth) [reply core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java]: Yep, now that it's published 😆

**Pull request merged**